### ISSUE TITLE
Some questions about the dataset

### ISSUE THREAD

**Issue by Judioljuse** [WhatAShot/Electrocardio-Panorama#12]
Hello, I would like to ask how the train_ptb.pkl file is constructed. I found that both the tianchi dataset and the ptb-xl dataset on physionet have a length of 5000. The processing on the tianchi dataset is to select a length of 512 (one and a half to two ECG cycle), train_ptb.pkl is a complete single cycle, and complete to 512 length. Is train_ptb.pkl constructed using the PTB-XL dataset on physionet, and how does it separate the ECG cycles? I think the training effect of ptb data is better than tianchi data set, is this related to the single cycle division of train_ptb.pkl?

**Comment by Judioljuse** [comment]
Hello, I found a small error in your paper, in the fifth part of Experiments, the angle of the V3 lead is wrong in the description of the dataset.
![1648870418(1)](https://user-images.githubusercontent.com/42485460/161364394-8d2a94f1-c98c-4049-96d1-4e928f9c8d50.png)

**Comment by WhatAShot** [comment]
Thank you for your finding. We will correct it soon.

**Comment by WhatAShot** [comment]
The cycle divisions for PTB were obtained using a segmentation model trained on Tianchi dataset, but we found the automatic cycle division annotations (for PTB) looked good and thus we did not further manually annotate cycle divisions of PTB.

For the phenomenon that the Nef-Net performances of PTB looks relatively worse than those of Tianchi, I considered that it seems unlikely that this phenomenon came from the quality of the cycle division annotations, as the division looked good. On the other hand, it seems not fair to compare the performances on two datasets directly, which can be somewhat affected by the data distributions or some other factors.
